.travis.yml

```diff
@@ -3,6 +3,7 @@ language: node_js
 node_js:
   - "8"
   - "9"
```

Comment:
remove 9 and add 11 instead?

Reply:
11 is already covered by `node`, I'd remove 9 though.

Reply:
I typically consider dropping support for a release of Node a semver-major act.